subworkflows/nf-core/mmseqs_fasta_cluster/tests/main.nf.test

```diff
@@ -0,0 +1,116 @@
+nextflow_workflow {
+
+    name "Test Subworkflow MMSEQS_FASTA_CLUSTER"
+    script "../main.nf"
+    workflow "MMSEQS_FASTA_CLUSTER"
+    config './nextflow.config'
+
+    tag "subworkflows"
+    tag "subworkflows_nfcore"
+    tag "subworkflows/mmseqs_fasta_cluster"
+    tag "mmseqs/createdb"
+    tag "mmseqs/cluster"
+    tag "mmseqs/linclust"
+    tag "mmseqs/createtsv"
+
+    test("fasta - linclust") {
+
+        when {
+            workflow {
+                """
+                input[0] = Channel.of([
+                    [ id: 'test_linclust' ],
+                    file(params.modules_testdata_base_path + '../../proteinfamilies/test_data/mgnifams_input_small.fa', checkIfExists: true)
+                ])
+                input[1] = 'linclust'
+                """
+            }
+        }
+
+        then {
+            assertAll(
+                { assert workflow.success},
+                { assert snapshot(
+                    file(workflow.out.clusters[0][1]).name,
+                    path(workflow.out.clusters[0][1]).readLines().size(),
+                    workflow.out.versions.collect { path(it).yaml }.unique()
+                    ).match() }
+            )
+        }
+    }
+
+    test("fasta - cluster") {
+
+        when {
+            workflow {
+                """
+                input[0] = Channel.of([
+                    [ id: 'test_cluster' ],
+                    file(params.modules_testdata_base_path + '../../proteinfamilies/test_data/mgnifams_input_small.fa', checkIfExists: true)
+                ])
+                input[1] = 'cluster'
+                """
+            }
+        }
+
+        then {
+            assertAll(
+                { assert workflow.success},
+                { assert snapshot(
+                    file(workflow.out.clusters[0][1]).name,
+                    path(workflow.out.clusters[0][1]).readLines().size(),
+                    workflow.out.versions.collect { path(it).yaml }.unique()
```

Comment:
Its good, but if you feel like it, maybe nft-csv plugin, & sort the entries? 
https://github.com/nf-core/modules/blob/620300d96c4b9051d533a5ea5f9d3b2c64b781ce/modules/nf-core/ska/distance/tests/main.nf.test#L33